### ISSUE TITLE
A11y_AzureToolsStorageExplorer_ToggleExplorerPaneTreeView_ScreenReader: VoiceOver announce the tree view an "empty table

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#2310]
GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #MAS4.2.1; #VoiceOver; #Mac; #CELAAssessID_CosmosDB-FY20-Q2;

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.11.0
MAC Version: Catalina 10.15 

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to "Open Connect dialog" pane and Sign in to Azure.
3. Go to "Manage Accounts" pane and Select the subscriptions you will use. Once selected, click on "Apply" button.
4. Start Voice Over.
5. Move into the tree view with VoiceOver navigation keys "Control+Option+RightArrowkey".
6. Observe VoiceOver announcement.

**​Actual:​**
VoiceOver announces the tree view an "empty table", Navigation works with up and down arrow keys as expected, but the name given the control by VoiceOver is still confusing.

**Expected:**
The control should be called something more appropriate, such as a tree view, or at any rate described as containing content.

**UserImpact:**
A VoiceOver user will be confused when the Explorer tree view is misidentified, and not expect to be able to navigate it.

**Comment by MRayermannMSFT** [comment]
I've merged a fix which adds a bit more description of where the focus is while navigating around the tree view. This is the *same* as what VS Code does. If testers are still not happy after verifying the fixes next month, then we'll need to file a bug against Chromium. There doesn't seem to be a way to remove the empty table dialog from voice over.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0 in MAC Version: Catalina 10.15, this issue is still reproduced.

**Comment by MRayermannMSFT** [comment]
Welp. I cannot find a good reference for an accessible tree that doesn't do this, and I don't want to do some sort of hack that ends up breaking windows. I can also confirm this bug happens in VS Code (which has a very similar tree view), and it also happens if I just use the reference tree view proved by W3C. I've opened a bug against chromium to hopefully get this issue addressed.

https://bugs.chromium.org/p/chromium/issues/detail?id=1048770

**Comment by MRayermannMSFT** [comment]
Chromium has accepted the bug. Will go over this with accessibility team on monday.

**Comment by Alexandria-R** [comment]
This appears to be external to Voiceover.

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#A11ySev2;

**Comment by BhanuPrakashGit** [comment]
GithubTag:#Regressed:06-01-23;#Closed;
Verified the bug in the environment:
Microsoft Azure Storage Explorer
Version 1.29.2

Bug is fixed and does not repro. Hence closing the bug.
PFA for reference
[2310_Closed.zip](https://github.com/microsoft/AzureStorageExplorer/files/11624835/2310_Closed.zip)
